### PR TITLE
Update autoscale var description

### PR DESCRIPTION
**Pull request by SirSpidey**
### Description

Add context to auto_scaling variable

#### No release required

- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other changes that don't affect Terraform code

#### Release required

- [ ] Bug fix (patch release (`x.x.X`): Change that fixes an issue and is compatible with earlier versions)
- [ ] New feature (minor release (`x.X.x`): Change that adds functionality and is compatible with earlier versions)
- [ ] Breaking change (major release (`X.x.x`): Change that is likely incompatible with previous versions)

##### Release notes content

Replace this text with information that users need to know about the bug fixes, features, and breaking changes. This information helps the merger write the commit message that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Merge by using "Squash and merge".
- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author.

    The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).

### CODE REVIEW

**Comment by SirSpidey** [comment]
@daniel-butler-irl: ready for your review.

File: examples/autoscale/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
...
 
-An end-to-end example that uses the module's default variable values. This example uses the IBM Cloud terraform provider to:
+An end-to-end example that uses the module's default variable values. This example uses the IBM Cloud Terraform provider to create the following infrastructure:
 
-- Create a new resource group if one is not passed in.
-- Create a new ICD Postgresql database instance with auto-scaling (automatically increase resources) enabled.
+- A resource group if one is not passed in.
+- An ICD Postgresql database instance with autoscaling (automatically increase resources) enabled.
+
+:information_source: **Tip:** Make sure you understand the effects of autoscaling, especially for production environments. See https://ibm.biz/autoscaling-considerations in the IBM Cloud Docs.
```
Comment (daniel-butler-irl) [inline examples/autoscale/README.md:8]: Not sure if we have one but should this be a warning tag? I don't mind if its an info tag though

Comment (SirSpidey) [reply examples/autoscale/README.md]: you have a point. updated to **Important**

**Comment by SirSpidey** [comment]
Closed in favor of #185

**Pull request closed without merging**